### ISSUE TITLE
Syntax error in react-konva.d.ts

### ISSUE THREAD

**Issue by sigmasoldi3r** [konvajs/react-konva#229]
Expected behavior: Compile with no error.

Versions:
```
typescript 2.6.2
yarn 1.7.0
Node 8.4.0
react-konva latest
react 16.2.0
```

What happened: TS Throws error at provided lines (94, 98, 99, 100)

Error (At compile time)
```
ERROR in C:\project_folder\node_modules\react-konva\react-konva.d.ts
[tsl] ERROR in C:\project_folder\node_modules\react-konva\react-konva.d.ts(94,3)
      TS1005: ',' expected.

ERROR in C:\project_folder\node_modules\react-konva\react-konva.d.ts
[tsl] ERROR in C:\project_folder\node_modules\react-konva\react-konva.d.ts(98,3)
      TS1005: ',' expected.

ERROR in C:\project_folder\node_modules\react-konva\react-konva.d.ts
[tsl] ERROR in C:\project_folder\node_modules\react-konva\react-konva.d.ts(99,56)
      TS1005: ',' expected.

ERROR in C:\project_folder\node_modules\react-konva\react-konva.d.ts
[tsl] ERROR in C:\project_folder\node_modules\react-konva\react-konva.d.ts(100,56)
      TS1005: ',' expected.
```

Example of line causing error:
```ts
export var Group: KonvaContainerComponent<Konva.Group> {}
```

Let me know if I can provide additional data or help.

**Comment by sigmasoldi3r** [comment]
Seems that this #230 might fix it

**Comment by lavrton** [comment]
Published `v1.7.12`
